zcash_client_sqlite/src/testing.rs

```diff
@@ -286,6 +289,57 @@ where
             limit,
         )
     }
+
+    pub(crate) fn get_total_balance(&self, account: AccountId) -> NonNegativeAmount {
+        get_wallet_summary(&self.wallet().conn, 0)
+            .unwrap()
+            .unwrap()
+            .account_balances()
+            .get(&account)
+            .unwrap()
+            .total()
+    }
```

Comment:
```suggestion
    /// Returns the sum of balances in all pools with the given `min_confirmations`, for the given account.
    pub(crate) fn get_total_balance_at_depth(&self, account: AccountId, min_confirmations: u32) -> NonNegativeAmount {
        get_wallet_summary(&self.wallet().conn, min_confirmations)
            .unwrap()
            .unwrap()
            .account_balances()
            .get(&account)
            .unwrap()
            .total()
    }

    /// Returns the sum of balances in all pools with no constraint on confirmations, for the
    /// given account.
    pub(crate) fn get_total_balance(&self, account: AccountId) -> NonNegativeAmount {
        self.get_total_balance_at_depth(account, 0)
    }
```
(may need reformatting).

Reply:
Note to self: when rebasing #908 on this, don't add `TestState::get_balance`, and replace uses of it with `TestState::get_total_balance`. Keep `TestState::get_balance_at`.

Reply:
This can be added when it's needed for a test.